common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -2,43 +2,37 @@
 metadata:
     display_name: Checkboxes
     markdown: |
-       Checkbox problems allow learners to select multiple options. Learners can see all the options along with the problem text.
+        You can use this template as a guide to the simple editor markdown and OLX markup to use for checkboxes problems. Edit this component to replace this template with your own assessment.
 
-       When you add the problem, be sure to select Settings to specify a Display Name and other values that apply.
+        >>Add the question text, or prompt, here. This text is required.||You can add an optional tip or note related to the prompt like this. <<
 
-       You can use the following example problem as a model.
+        [x] a correct answer
+        [ ] an incorrect answer
+        [ ] an incorrect answer
+        [x] a correct answer
 
-       >>The following languages are in the Indo-European family:||Make sure you select all of the correct options—there may be more than one!<<
-       [x] Urdu
-       [ ] Finnish
-       [x] Marathi
-       [x] French
-       [ ] Hungarian
-
-       [explanation]
-       Urdu, Marathi, and French are all Indo-European languages, while Finnish and Hungarian are in the Uralic family.
-       [explanation]
+        [explanation]
+        You can specify an optional explanation that appears after the selected answers are submitted.
+        [explanation]
```

Comment:
Thanks @muzaffaryousaf ! (I previously commented on the wiki page, I do apologize for the inconsistency. I agree that it is best to show it for all problem types, or none. Thanks!)